src/App.js

```diff
@@ -28,6 +28,7 @@ export default function App() {
         <Route path="/action" component={Action} />
         <Route path="/inscription" component={Inscription} />
         <Route path="/action" component={ActionPage} />
+        <Route path="/BDDFavart" component={Link} />
```

Comment:
`<Link>` being a named component from `react-router`, I **really** recommend that you change this name (and once again, why mention the database at all ?)